packages/server/dataloader/azureDevOpsLoaders.ts

```diff
@@ -347,6 +349,45 @@ export const allAzureDevOpsProjects = (
   )
 }
 
+export const azureDevOpsProject = (
+  parent: RootDataLoader
+): DataLoader<AzureDevOpsRemoteProjectKey, AzureDevOpsRemoteProject | never[], string> => {
+  return new DataLoader<AzureDevOpsRemoteProjectKey, AzureDevOpsRemoteProject | never[], string>(
```

Comment:
Thanks for the explanation, that's helpful! I've updated the type. The `never[]` type happened because the `return results.map(...` line had an empty array in the ternary. I've changed that to `null` and removed the `never[]`